app/models/spree/payment.rb

```diff
@@ -145,7 +155,7 @@ def adjustment_label
       I18n.t('payment_method_fee')
     end
 
-    def mark_as_processed
+    def clear_authorization_url
```

Comment:
:+1:

Reply:
nice! we could add a specific field in the db for this (that was my original idea at least).
payment.authorization_url